site/docs/visual_elements/favicon.mdx

```diff
@@ -0,0 +1,161 @@
+
+**Do not attempt to recreate the favicon!** 
+Always use the appropriate original file. 
+
+**Note:** The City of Helsinki logo is a registered trademark.
+
+## Principles 
+Official City of Helsinki services must always show the City of Helsinki favicon: 
+- sitewide, in the browser website title
+- use the same favicon variant consistently
+
+## Usage
+
+### Contrast
+The Helsinki framed favicon is primarily used in black, white, or with a transparent background depending on the circumstance.
+- Check that the colour contrast of the favicon and its background is compliant with <Link href="https://www.w3.org/TR/WCAG21/#contrast-minimum" external>AA Level contrast ratios</Link>.
+- For more information, see the <Link href="https://brand.hel.fi/en/colours/" external>Visual Identity Guidelines - Colours</Link>.
+
+### Download
+
+<p id="github">The favicon of City of Helsinki digital services is a minimised version of the Helsinki logo. Different favicons are available as a downloadable ZIP package.</p>
+<p>You can <Link href="https://github.com/City-of-Helsinki/helsinki-design-system/releases/latest/download/hds-favicon-kit.zip" external>download the latest HDS Favicon kit from the HDS GitHub repository</Link>.</p>
+
+## Installation
+First, use the download link to download the files listed below. Unzip the archive to the root directory of your website.
+
+```android-chrome-192x192.png```
+
+```android-chrome-512x512.png```
+
+```apple-touch-icon.png```
+
+```favicon-16x16.png```
+
+```favicon-32x32.png```
+
+```favicon.ico```
+
+```site.webmanifest```
+
+Next, copy the following link tags and paste them into the `head` of your HTML.
+```html
+
+<link rel="apple-touch-icon" sizes="180x180" href="/apple-touch-icon.png">
+
+<link rel="icon" type="image/png" sizes="32x32" href="/favicon-32x32.png">
+
+<link rel="icon" type="image/png" sizes="16x16" href="/favicon-16x16.png">
+
+<link rel="icon" type="image/ico" sizes="16x16" href="/favicon.ico">
+
+<link rel="manifest" href="/site.webmanifest">
+
+```
+
+## Variants
+There are many variants available in HDS. They all serve a very specific purpose. 
+
+<p> <span style="font-weight: bold;">There is no need to download the icons one-by-one. You can download the whole favicon package from the</span> <Link href="#github"internal>Link above ↑</Link>.</p>
+
+
+### Light favicons
+
+Favicon variant                                                                                                                               | Filename          | Format                        |
+--------------------------------------------------------------------------------------------------------------------------------------------- |-------------------|-------------------------------|
+![Light 16×16px favicon](../../static/favicon/thumb/light_favicon_16x16.png)                                                                  | favicon-16x16.png | .png                          |
```

Comment:
Now that we only have one image per table here, maybe we could just ditch the images altogether? We can either provide a list (table) of all favicon variants or just leave the table out. Since the user cannot download images via this table anymore, I do not know if there is much value added here.

Reply:
Check ✅